dojo/api.py

```diff
@@ -89,7 +95,85 @@ def form_args(self, bundle):
         return kwargs
 
 
+def sanitize(text):
```

Comment:
shouldn't this be in a helper of sort?

Reply:
It's just for this apiv1 thing that is going to be thrown away in june

Reply:
ok